test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -266,20 +268,21 @@ private async Task<Mock<RequestContext<DacFxResult>>> SendAndValidateGenerateDep
             var generateScriptParams = new GenerateDeployScriptParams
             {
                 PackageFilePath = extractParams.PackageFilePath,
-                DatabaseName = string.Concat(sourceDb.DatabaseName, "-deployed"),
-                ScriptFilePath = Path.Combine(folderPath, string.Concat(sourceDb.DatabaseName, "_", "UpgradeDACScript.sql"))
+                DatabaseName = targetDb.DatabaseName
             };
 
+            // Generate script for deploying source dacpac to target db
             GenerateDeployScriptOperation generateScriptOperation = new GenerateDeployScriptOperation(generateScriptParams, result.ConnectionInfo);
             service.PerformOperation(generateScriptOperation);
```

Comment:
why no TaskExecutionMode.Script here?

Reply:
It doesn't matter since the tests don't create s SqlTask, but I'll change this since it's unclear.